srcpkgs/clipmenu/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'clipmenu'
+pkgname=clipmenu
+version=5.6.0
+revision=1
+noarch=yes
+create_wrksrc=yes
```

Comment:
Without `create_wrksrc=yes`, `$pkgname-$version/` in do_install will not be needed.